src/mpi/errhan/errnames.txt

```diff
@@ -938,6 +938,12 @@ is too big (> MPIU_SHMW_GHND_SZ)
 **mpl_gavl_create: mpl_gavl_create failed
 **mpl_gavl_delete_range: mpl_gavl_delete_range failed
 **mpl_gavl_delete_start_addr: mpl_gavl_delete_start_addr failed
+
+## Async thread affinity related error messages
+**parse_thread_affinity:Failed to parse async thread affinity string
+**parse_thread_affinity %s:Failed to parse async thread affinity string [%s]
+**set_thread_affinity:Failed to set the async thread affinity
```

Comment:
Can we add a long message for this as well? For example, show what affinity is the process trying to set that failed.

Reply:
With the current code structure, we read and parse the affinity string in `get_thread_affinity/MPIDI_parse_progress_thread_affinity`. If we were to print the affinity string string, we may need to return it from these functions to `MPIR_Init_async_thread` or directly read from `MPIR_Init_async_thread`. Thoughts on if we go on any of such routes?

Reply:
This is the error for `MPL_thread_set_affinity(progress_thread_id, &(thread_affinity[affinity_idx]), 1, &thr_err);`, can we include `thread_affinity(affinity_idx)` in the error message so user would have some clue why it failed?

Reply:
Done.